src/main/scala/edu/ie3/simona/model/participant/PvModel.scala

```diff
@@ -478,6 +485,8 @@ final case class PvModel private (
 
         val omega1InRad = omega1.toRadians
         val omega2InRad = omega2.toRadians
+        // variable that accounts for cases when the integration interval is shorter than 15° (1 hour equivalent), when the time is close to sunrise or sunset
+        val timeFrame = (omega2 - omega1).toDegrees / 15d / duration.toHours // original term: (omega2 - omega1).toRadians * 180 / Math.PI / 15, since a one hour difference equals 15°
```

Comment:
I'm not fully convinced by your proposal yet. 

The original problem is that the ratio between `a` and `b` can tend to infinity because the radiation on the horizontal surface goes to 0, while the radiation on tilted surface can stay fairly big if tilted towards sunset.

The solution proposed with `R_{b,ave} = a/b` is to calculate some sort of average over a time period between omega1 and omega2. This way, the extreme values calculated when using just one point in time close to sunrise or sunset gets reduced. This works because the value further away from sunrise or -set pulls down the average again. 

All this was just how it's supposed to work. If now the time slice between omega1 and omega2 is very small, the "relief" we get from the second value is quite reduced. Now it seems like some sort of "crutch on a crutch" or "workaround around a workaround" to try to calculate our way back to the time interval of one hour. This is because calculating an average from a time interval is already a workaround (which could be debatable, I guess). And furthermore this would only work if the change in radiation is _proportional_ to the change in angle omega, which doesn't seem clear to me.

Curious about your thoughts.

Reply:
I am not quite sure what you mean by "calculate our way back to the time interval of one hour". If the difference between omega1 and omega2 is very small, we are not looking at a full hour, but rather a few minutes. The problem is that the available weather data contains the radiation energy of one full hour in Wh/m^2, which would obviously result in a false result if we multiply it by R_b,ave. This is why there is the linear factor "timeFrame" between 1 and 0, which should solve this problem. 

Also, I partially get what you mean by "workaround around a workaround", but doesn't that also apply for the current implemenation aswell? There, the problem is solved by artificially keeping the difference between omega1 and omega2 fixed, which is inaccurate at times close to sunrise or sunset. 

Besides, can you please explain why the change in radiation must be proportional to the change in omega?

Furthermore, if you already looked at the integration test, did you notice anything odd? Were there any very high peaks?

Reply:
It's actually proportional to the timeframe duration, which is itself proportional to the angle though.

Reply:
I rather meant "proportional to the timeframe duration", but that's proportional to the angle anyways.